example/viewModeChangeExample.js

```diff
@@ -0,0 +1,34 @@
+var DateTime = require('../DateTime.js');
+var React = require('react');
+var ReactDOM = require('react-dom');
+var moment = require('moment');
+
+var Wrapper = React.createClass({
+	getInitialState: function() {
+		return {
+			viewMode: 'time'
+		};
+	},
+
+	updateView: function(format) {
+		console.log('changing state');
+		this.setState({
+			viewMode: 'days'
+		});
+	},
+
+	componentDidMount: function() {
+		setTimeout(this.updateView, 3000);
+	},
+
+	render: function() {
+		console.log(this.state.viewMode);
```

Comment:
Why is this log still here? If you think it's useful maybe do it like

`console.log('Current viewmode: ' + this.state.viewMode);`